Blockzilla/BrowserViewController.swift

```diff
@@ -281,6 +281,8 @@ class BrowserViewController: UIViewController {
         alertController.addAction(UIAlertAction(title: UIConstants.strings.copyImage, style: .default) { _ in copyAction() })
         alertController.addAction(UIAlertAction(title: UIConstants.strings.cancel, style: .cancel))
 
+        alertController.popoverPresentationController?.sourceView = self.view
+        alertController.popoverPresentationController?.sourceRect = CGRect(x: self.view.bounds.size.width / 2.0, y: self.view.bounds.size.height / 2.0, width: 1.0, height: 1.0)
```

Comment:
Do we always want the menu to save images to show up in the center? If not, we can find some like sort of anchor point (e.g. like the touch point) and do something like below.. otherwise, I'm good with just centering it in the middle (which I think is what you're doing here)

`alertController.popoverPresentationController?.sourceView = anchor
  alertController.popoverPresentationController?.sourceRect = anchor.bounds
  alertController.popoverPresentationController?.permittedArrowDirections = .up`

Reply:
Yeah, I started to go down that route. But because its in the WebView it becomes a little bit trickier than I thought. Might be outside the bounds of this fix? I'll open a ticket for it though.